sources/collections/tests/bit-count.dylan

```diff
@@ -62,7 +62,7 @@ define test bit-count-multiple-word-sized-vector ()
     bit-count(vector, bit-value: 1), size(bits));
 end test;
 
-define suite bit-count-suite (description: "Test bit-count")
```

Comment:
This doesn't compile, does it?  I believe you at least need the `()` to be there.

Reply:
Pretend I put that comment after the following line. Still not used to GitHub review UI.

Reply:
Fixed, thanks.